### PR TITLE
Add documentation for the sandbox

### PR DESCRIPTION
**Pull request by christophermluna**
- create pages documenting the MaxMind Sandbox environment
- fix copy on minFraud Legacy warning to indicate that changes have already occurred

### CODE REVIEW

File: content/geoip/sandbox-environment.mdx
Comment (dhogan8) [inline content/geoip/sandbox-environment.mdx:1]: This file path doesn't match the URL specified (`/geoip/docs/web-services/sandbox-environment`).

File: content/minfraud/sandbox-environment.mdx
Comment (dhogan8) [inline content/minfraud/sandbox-environment.mdx:66]: It is not clear to me why this is a level 2 heading, but everything above is a level 3. I think they should probably all be level 2. This is also contributing to one of the build failures.

File: content/geoip/sandbox-environment.mdx
Comment (dhogan8) [inline content/geoip/sandbox-environment.mdx:14]: I think these should all be level 2 headings.

File: content/minfraud/sandbox-environment.mdx
Comment (dhogan8) [inline content/minfraud/sandbox-environment.mdx:10]: There seems to be a lot of duplicate content for the minFraud and GeoIP sandbox files. It might make sense to condense this and use a shared template for the similar parts. This would save others time in the future to only have to update one file rather than updating two with the same exact content.

**Review by dhogan8 — DISMISSED** [review]

**Comment by dhogan8** [comment]
I also recommend rebasing off of `main` since it has the precommit hook to lint any files you update or create. Alternatively, I recommend running `npx prettier --write <filename>` so that Prettier can fix the files.

**Comment by christophermluna** [comment]
i rebased off main, made all the headers level 2, and updated the documentation to reflect what was discussed about test data

**Comment by christophermluna** [comment]
i want to make sure we look back at @dhogan8's comment about taking duplicate content and putting it into a template when i make final changes

**Comment by christophermluna** [comment]
so the build failure is a [broken link issue](https://github.com/maxmind/dev-site/runs/8235321330?check_suite_focus=true#step:5:28), which i think may be because the link is pointing to the account portal or the sandbox and for some reason that's coming back as a broken link.

**Comment by dhogan8** [comment]
> so the build failure is a [broken link issue](https://github.com/maxmind/dev-site/runs/8235321330?check_suite_focus=true#step:5:28), which i think may be because the link is pointing to the account portal or the sandbox and for some reason that's coming back as a broken link.

@christophermluna  This commit should do the trick https://github.com/maxmind/dev-site/pull/776/commits/24f85229bae67af6c92fbe6f0ad0d0993b7fdc47.

**Comment by christophermluna** [comment]
i have fixed the issue with templates and have rebased against main due to the e2e test that was failing. waiting to get confirmation that everything is working as expected with the minFraud sandbox data in production. don't merge without checking in.

**Comment by dhogan8** [comment]
@christophermluna A few notes:
- Please see my comment above about the failed `Staging / Test - End-to-end` check. You can probably git cherry pick that or add it in as a separate commit.
- When looking at [minFraud Sandbox](https://mm-dev-site-staging--pr-727-bjx2ccwa.web.app/minfraud/sandbox-environment?lang=en#learn-more-about-the-maxmind-sandbox) and [GeoIP Sandbox](https://mm-dev-site-staging--pr-727-bjx2ccwa.web.app/geoip/docs/web-services/sandbox-environment?lang=en#learn-more-about-the-maxmind-sandbox) there is a duplicate section.
- If a PR is not ready to be merged, it is helpful to mark the [PR as a draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft) until it is ready for review and merge.

File: content/minfraud/sandbox-environment.mdx
Comment (christophermluna) [inline content/minfraud/sandbox-environment.mdx:55]: @dlieou are these the correct expected responses? i was looking at your discussion of the test data and i think i must have pulled the wrong numbers.

File: content/geoip/docs/web-services/sandbox-environment.mdx
Comment (dlieou) [inline content/geoip/docs/web-services/sandbox-environment.mdx:10]: What would lines 6 to 9 look like in a doc?

File: content/geoip/docs/web-services/sandbox-environment.mdx
Comment (dlieou) [inline content/geoip/docs/web-services/sandbox-environment.mdx:50]: FWIW, I tested out all the 3 endpoints on lines 31 to 33 using the file mentioned on line 49, and using the first IP in it, both city and insights returned correct info. For the country one, I got {"code":"IP_ADDRESS_NOT_FOUND","error":"The IP address '::17c:d500' is not in our database."

The file for country is https://github.com/maxmind/MaxMind-DB/blob/main/source-data/GeoIP2-Country-Test.json

File: content/minfraud/sandbox-environment.mdx
Comment (dlieou) [inline content/minfraud/sandbox-environment.mdx:34]: These endpoints are correct. I also tested them out and got the results back.

File: content/minfraud/sandbox-environment.mdx
Comment (dlieou) [reply content/minfraud/sandbox-environment.mdx:55]: @christophermluna this is where we still have a discrepancy. Those are the numbers I am getting. The good news is whether I entered these IPs in the query form or via curl command, the results I am getting for each IP are the same. With that said, each day I tried the results seemed to be different (went down a bit). :(

Comment (christophermluna) [reply content/minfraud/sandbox-environment.mdx:55]: what i am asking is, i thought you guys were talking about a "medium" score, but in this test data there's just a high IP risk score and two low IP risk scores. is that correct? is it correct that two of the IPs should return a near-0 IP risk score, and one of them should return a value near 75?

File: content/geoip/docs/web-services/sandbox-environment.mdx
Comment (christophermluna) [reply content/geoip/docs/web-services/sandbox-environment.mdx:10]: lines 6-9 import the content listed in those lines when the call is made in the document

so, for example, line 11 is:

`<SandboxIntro />`

this references line 6, where we have imported `/content/sandbox-intro.mdx`.

so line 11 in this file would be replaced with the full contents of `/content/sandbox-intro.mdx`.

you can see how this looks in practice with the preview of the site:

https://mm-dev-site-staging--pr-727-bjx2ccwa.web.app/geoip/docs/web-services/sandbox-environment?lang=en

**Review by its-johnt — APPROVED** [review]

**Pull request merged**
